docs/README.md

```diff
@@ -5,6 +5,9 @@
   * [Tutorial](/docs/tutorial/Tutorial.md)
   * [Testing](/docs/tutorial/Testing.md)
   * [Advanced topics](/docs/tutorial/AdvancedTopics.md)
+  * [Recipes](/docs/tutorial/Recipes.md)
```

Comment:
@on3iro you missed this link. It's not pointing to a file so it's grayed out. It can just be removed